src/IQFeed.CSharpApiClient/Utils/TradeConditionsLookupTable.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using IQFeed.CSharpApiClient.Lookup;
+using IQFeed.CSharpApiClient.Lookup.Symbol.Messages;
+
+namespace IQFeed.CSharpApiClient.Utils
+{
+    public class TradeConditionsLookupTable
+    {
+        private static readonly ITradeCondition[] NoTradeConditions = new ITradeCondition[0];
+
```

Comment:
not a fan of these util classes, maybe we can add them in the extensions eventually.
could you please remove them?

Reply:
sure, I thought this will be reusable for other too, I can move it to my codebase